requirements.txt

```diff
@@ -1,4 +1,6 @@
 cryptography>=0.9.2
+enum34>=1.0.4
+ordereddict>=1.1
```

Comment:
These need to be removed, they break the build on Python 3.5. I removed them in the commit I merged to master.